### DISCUSSION TITLE
Thank you so much!

### DISCUSSION THREAD

**Discussion by hruzgar** [remotely-save/remotely-save#84] [General]
I just wanted to thank you guys for making this plugin. You really are helping me out. Currently i am a college-student and cannot afford the monthly fee for the official sync service. Thank you really much! I really really appreciate your work! You guys are a true gems.
Hope you'll have a nice day
(sorry for my bad english)

**Comment by brianmassen** [comment]
Could I just throw in my vote of thanks for this plugin it is a life saver. I have a desktop and two laptops synced through my Synology nas. I also have an iPad. The only apparent options for syncing Obsidian from the iPad are the paid option or iCloud. 
I have now managed to enable WebDAV on my nas and sync my iPad using your plugin. Brilliant.

many thanks yet again.

just a quick edit.

The plug-in synced incorrectly today, instead of updating the iPad with all the work I’d done on my desktop it decided to copy some files from the nas to the iPad but also overwrote all my work on the nas with older files from the iPad. Which I’m afraid means I cannot trust the sync.

I’ve now disabled the plug-in for now, I may have another look at it in the future.

**Reply by berot3** [reply]
Me too!

iOS sux, but this is what I am stuck on, for now...

This plugin finally allows me to choose something else but this very limited iCloud! 

💜
